.github/workflows/ci_codeql.yml

```diff
@@ -24,8 +24,6 @@ jobs:
     name: Analyze by CodeQL
     runs-on: ubuntu-latest
     permissions:
-      actions: read
-      contents: read
```

Comment:
These two fields are only required for workflows in private repositories. So I deleted them.

Reply:
change back to `disable-default-queries: false`  ?